decisions.md

```diff
@@ -30,3 +30,12 @@ Certain actions require a vote of two thirds of the members, rather than a simpl
 
 * Removal of a Core Team member
 * Removal of the Core Team Chair
+
+## Changes to the Governance Documents
+
+The governance documents for the NUnit Community are stored in the [GitHub nunit/governance](https://github.com/nunit/governance) repository. Any changes to those documents must go through a Pull Request and be reviewed by the Core Team. Pull requests may be submitted by anyone in the community, but changes may only be merged by members of the Core Team.
+
```

Comment:
😂 I was just reviewing the 'revised' version, and was about to suggest the point about spelling fixes should be re-worded for the situation where non-Core Team members may submit pull requests. Never mind!